sktime/forecasting/trend/_polynomial_trend_forecaster.py

```diff
@@ -151,6 +160,71 @@ def _predict(self, fh=None, X=None):
         y_pred.name = self._y.name
         return y_pred
 
+    def predict_interval(self, fh, coverage=[0.95]):
+        """Computes the prediction intervals for different confidence levels"""
+        import numpy as np
+        from scipy.stats import norm
+        
+        def l_days_since_1970(idx):
```

Comment:
I think this duplicates `_get_X_numpy_int_from_pandas` and might use a different unit of time?